code/modules/clothing/shoes/boots.dm

```diff
@@ -7,7 +7,10 @@
 	item_state = "jackboots"
 	force = 3
 	armor = list(
-		melee = ARMOR_MELEE_KNIVES
+		melee = ARMOR_MELEE_KNIVES,
+		energy = ARMOR_ENERGY_MINOR,
+		bomb = ARMOR_BOMB_MINOR,
+		rad = ARMOR_RAD_MINOR
```

Comment:
There is no reason why security should have rad armor on their boots

Reply:
Would it be best to remove rad armor from both; equalize armor values by removing the extra armor values from the workboots instead of adding them to jackboots; just remove the rad armor from jackboots; or balance the two boot types with an equal number of armor types but different types (ie: minimum bullet armor for jackboots instead of rad armor, or something like that)?